src/lib/containers/widgets/Checkbox.tsx

```diff
@@ -49,14 +51,15 @@ export const Checkbox: React.FunctionComponent<CheckboxProps> = (
   return (
     <div className={className} onClick={props.onClick}>
       <input
+        aria-label={props.label}
         type="checkbox"
         checked={checked}
         id={uniqueId}
         onChange={handleCheckboxChange}
         disabled={disabled}
         data-testid={props['data-testid']}
       />
-      <label htmlFor={uniqueId}>{props.label}</label>
+      {<label htmlFor={uniqueId}>{hideLabel ? <></> : props.label}</label>}
```

Comment:
Is a react fragment preferable to an empty string here?

Reply:
Not sure if it matters, I'm trying not to imply that the label is the empty string. Unfortunately, removing the label breaks our checkbox styling.

Reply:
Not suggesting to remove the label, just asking about the label text ('' vs <>/<>)